docs/UserGuide.md

```diff
@@ -214,7 +214,7 @@ The following parameters are used for commands:
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `ac n/NAME`, `NAME` is a parameter which can be used as `ac n/John Doe`.
 
-* Items in square brackets are optional.<br>
+* Items in square brackets are optional. If an optional prefix's value is left blank, it is assumed that you are leaving the value blank or deleting the already existing value.<br>
```

Comment:
Feels repeated? What about "If an optional prefix's value is left blank, it is assumed that are deleting the already existing value."

Reply:
I think the current sentence is good, since for the add command, inputing `t/` will leave the value blank.

Reply:
I tried to modify it a bit to sound better - will be going with 

If an optional prefix is used without a value, it is assumed that you are creating a user without the value or deleting the already existing value.

Thanks for the advice!